demo/image-segmentation/README.md

```diff
@@ -59,11 +59,11 @@ We will spin up a [Ray cluster](https://docs.ray.io/en/latest/cluster/key-concep
 
 > **_NOTE:_**  It would be best to run the following steps on your workstation with a display device so you can see the Ray dashboard using a browser.
 
-1. Clone the `dataflux-pytorch` repository and go to the working directory.
+1. Clone the `dataflux-pytorch` repository, install the packages and go to the working directory.
    ```shell
    git clone --recurse-submodules https://github.com/GoogleCloudPlatform/dataflux-pytorch
    cd dataflux-pytorch
-   pip install .
+   pip install gcs-torch-dataflux
```

Comment:
Yes because all the training code is stored in demo/image-segmentation and is not installed with the package and that's intended since they are not part of the dataflux dataset library. Users will need to clone the repository to have access to the training code.